tests/tests_modifications_with_selinux_disabled.yml

```
@@ -0,0 +1,120 @@
+---
+- name: Ensure the default is targeted, enforcing, without local modifications
+  hosts: all
+  gather_facts: true
+  vars:
+    selinux_all_purge: true
+    selinux_logins_purge: true
+
+  tasks:
+    - name: Ensure SELinux tool semanage
+      package:
+        name:
+          - policycoreutils-python-utils
+        state: present
+      when: ansible_distribution == "Fedora" or
+        (ansible_distribution_major_version | int > 7 and
+         ansible_distribution in ["CentOS", "RedHat", "Rocky"])
+
+    - name: Ensure SELinux tool semanage on EL7
+      package:
+        name:
+          - policycoreutils-python
+        state: present
+      when:
+        - ansible_distribution_major_version | int == 7
+        - ansible_os_family == "RedHat"
+
+    - name: Ensure selinux-policy-targeted
+      package:
+        name:
+          - selinux-policy-targeted
+        state: present
+
+    - name: Add a Linux System Roles SELinux User
+      user:
+        comment: Linux System Roles SELinux User
+        name: sar-user
+    - name: Backup original /etc/selinux/config
+      copy:
+        remote_src: true
+        src: /etc/selinux/config
+        dest: /etc/selinux/config.test_selinux_disabled
+        mode: preserve
+    - name: Upload testing /etc/selinux/config
+      copy:
+        src: selinux.config
+        dest: /etc/selinux/config
+        mode: preserve
+    - name: Switch to permissive to allow login when selinuxfs is not mounted
+      command: setenforce 0
+      changed_when: false
```

```suggestion
      changed_when: true
```
because you are unconditionally changing the state of the system